command.go

```diff
@@ -1470,6 +1472,100 @@ func xGroupInfoParser(rd *proto.Reader, n int64) (interface{}, error) {
 
 //------------------------------------------------------------------------------
 
+type XInfoStreamCmd struct {
+	baseCmd
+	val XInfoStream
+}
+
+type XInfoStream struct {
+	Length          int64
+	RadixTreeKeys   int64
+	RadixTreeNodes  int64
+	Groups          int64
+	LastGeneratedID string
+	FirstEntry      XMessage
+	LastEntry       XMessage
+}
+
+var _ Cmder = (*XInfoStreamCmd)(nil)
+
+func NewXInfoStreamCmd(ctx context.Context, stream string) *XInfoStreamCmd {
+	return &XInfoStreamCmd{
+		baseCmd: baseCmd{
+			ctx:  ctx,
+			args: []interface{}{"xinfo", "stream", stream},
+		},
+	}
+}
+
+func (cmd *XInfoStreamCmd) Val() XInfoStream {
+	return cmd.val
+}
+
+func (cmd *XInfoStreamCmd) Result() (XInfoStream, error) {
```

Comment:
`Val` and `Result` should return `*XInfoStream`. `XInfoStream` is too big for returning a copy.